.github/workflows/kyverno.yaml

```
@@ -64,24 +66,21 @@ jobs:
         chart:
           - ${{ fromJson(needs.list-changed.outputs.charts) }}
     steps:
-      - name: Checkout
+      - name: Checkout repo
         uses: actions/checkout@v3
 
-      - uses: actions/checkout@v3
-        name: Checkout kyverno-policies
-        with:
-          repository: kyverno/policies
-          ref: main
-          path: kyverno-policies
-
       - name: Install Helm
         uses: azure/setup-helm@v3
 
```

To successful run the workflow the charts dependencies needs to be downloaded.
The best option is to do that with the `hack/helm-dependencies.bash` script:

```suggestion

      - name: Download chart dependencies (recursive)
        shell: bash
        run: hack/helm-dependencies.bash

```